méli-mélo/deprecated/deprecated.html

```diff
@@ -41,3 +41,71 @@ <h3>Code</h3>
 		&lt;/div&gt;
 	&lt;/div&gt;
 &lt;/div&gt;</code></pre>
+
+<h2>Follow us with Google+ icon <span class="label label-warning">Deprecated as of v16.0.0</span></h2>
+<section class="followus">
+	<h2>Follow:</h2>
+	<ul>
+		<li><a href="#facebook" class="facebook wb-lbx"><span class="wb-inv">Facebook</span></a></li>
+		<li><a href="#" class="twitter" rel="external"><span class="wb-inv">Twitter</span></a></li>
+		<li><a href="#youtube" class="youtube wb-lbx"><span class="wb-inv">YouTube</span></a></li>
+		<li><a href="#" class="flickr" rel="external"><span class="wb-inv">Flickr</span></a></li>
+		<li><a href="#" class="pinterest" rel="external"><span class="wb-inv">Pinterest</span></a></li>
+		<li><a href="#" class="linkedin" rel="external"><span class="wb-inv">LinkedIn</span></a></li>
+		<li><a href="#" class="instagram" rel="external"><span class="wb-inv">Instagram</span></a></li>
+		<li><a href="#" class="rss" rel="external"><span class="wb-inv">Fil RSS</span></a></li>
+		<li><a href="#" class="email" rel="external"><span class="wb-inv">Email subscription</span></a></li>
+		<li><a href="#" class="periscope" rel="external"><span class="wb-inv">Periscope</span></a></li>
+		<li><a href="#" class="googleplus" rel="external"><span class="wb-inv">Google+</span></a></li>
+	</ul>
+</section>
+
+<h2>Background image heading <span class="label label-warning">Deprecated as of v16.0.0</span></h2>
+<p>Add a background image to a heading.</p>
+
+<h3>Working example</h3>
```

Comment:
Remove the existing working example that we have in our current provisional page. And in the provisional page link to this deprecated page.